inhouse_bot/database_orm/tables/game.py

```diff
@@ -27,27 +27,26 @@ class Game(bot_declarative_base):
     __tablename__ = "game"
 
     # Auto-incremented ID field
-    id = Column(Integer, primary_key=True)
+    id: int = Column(Integer, primary_key=True)
 
     # Game creation date
-    start = Column(DateTime)
+    start: datetime = Column(DateTime)
 
     # Server the game was played from
-    server_id = Column(BigInteger)
+    server_id: int = Column(BigInteger)
```

Comment:
also BigInteger is long in python, I feel making it or expecting type int could probably bring problems in the future

Reply:
actually nvm, ints have no max value in python, but I feel like technically it should be long to give the user an idea on the length of the number

Reply:
https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.BigInteger